File: CompilationTest/FileTest/results/FileTest.pl
use strict;
use warnings;

open my $fh2, '>', "test.txt" or die "Can't open file $!";
print $fh2, "example content", -s $fh2;
close $fh2;;
open my $fh, '<', "test.txt" or die "Can't open file $!";
read $fh, my $file_content, -s $fh;
close($fh);
print(($file_content) . "\n");

File: CompilationTest/FileTest/results/FileTest.go
package main

import "fmt"
import "io/ioutil"


func main() {
    ioutil.WriteFile("test.txt", []byte("example content"), 0644)
    fileContentBytes, _ := ioutil.ReadFile("test.txt")
    fileContent := string(fileContentBytes)
    fmt.Println(fileContent)
}